src/fixtures.js

```diff
@@ -1,25 +1,29 @@
+/* global self */
 'use strict'
 
 const isNode = require('detect-node')
 const path = require('path')
-const Buffer = require('safe-buffer').Buffer
 
-module.exports = function loadFixtures (dirname, file, module) {
+module.exports = function loadFixtures (dirname, filePath, module) {
   if (isNode) {
-    return require('fs').readFileSync(path.join(dirname, file))
+    return require('fs').readFileSync(path.join(dirname, filePath))
   } else {
-    return syncXhr(file, module)
+    return syncXhr(filePath, module)
   }
 }
 
-// I know this is considered bad practice, but it makes testing life so much nicer!
-function syncXhr (url, module) {
+// @dignifiedquire: I know this is considered bad practice, but it makes testing life so much nicer!
+function syncXhr (filePath, module) {
   let target
   if (module) {
-    target = path.join('/base', 'node_modules', module, 'test', url)
+    target = path.join('/base', 'node_modules', module, filePath)
```

Comment:
Wouldn't removing `test` from the path break existing tests? I feel like I'm missing something.

Reply:
You are right. Did more testing and realized that there was even a better fix